src/EditorFeatures/Core/Interactive/InteractiveSession.cs

```diff
@@ -194,6 +226,12 @@ private void AddSubmissionProjectNoLock(ITextBuffer submissionBuffer, string lan
             _currentSubmissionProjectId = ProjectId.CreateNewId(newSubmissionProjectName);
             var newSubmissionDocumentId = DocumentId.CreateNewId(_currentSubmissionProjectId, newSubmissionProjectName);
 
+            // Associate a name with both the editor document and our roslyn document so LSP can make requests on it.
+            var newSubmissionFilePath = Path.Combine(_hostDirectory, $"Submission{SubmissionCount}.{_languageInfo.Extension}");
```

Comment:
>_languageInfo.Extension [](http://example.com/codeflow?start=100&length=23)

I think usually the extension contains the `.` (e.g. `Path.GetExtension("a.cs")` is ".cs". It'd be good to keep it that way and add a doc comment to the property that says so.

Reply:
yep missed that when I moved it, will update